root/Makefile

```
@@ -202,7 +202,7 @@ docker-build:: pre-docker-build
 	@echo " ===> building docker image <==="
 	@ssh-add -L
 	@echo " ===> If you run into credential issues, ensure that your key is in your SSH agent (ssh-add <ssh-key-path>) <==="
-	DOCKER_BUILDKIT=1 docker build --ssh default -t gcr.io/outreach-docker/$(APP) -f deployments/$(APP)/Dockerfile . --build-arg VERSION=${APP_VERSION}
+	@./shell/ci/release/docker.sh
```

issue: I actually tried this change out locally because I was having trouble building a docker image in my service, and I got the following error:

```
bash: line 1: ./shell/ci/release/docker.sh: No such file or directory
```

This is what worked for me to at least get the script running:
```suggestion
		@./scripts/shell-wrapper.sh ci/release/docker.sh
```

---

Thanks for testing this out. Wired, I tried this locally and works for me. Might be different indentation messed up? I use zsh. I will check the indentation again.

---

I updated to use shell-wrapper to run the script, indentation doesn't seem to matter for me. LMK if it still doesn't work for your environment.

---

Whoops, the indentation was a copy/paste error. Sorry about that.